src/main/clojure/clara/rules/compiler.clj

```diff
@@ -1697,11 +1796,9 @@
                              entry))
 
         ;; type, alpha node tuples.
-        alpha-nodes (for [{:keys [type alpha-fn children env] :as alpha-map} alpha-fns
+        alpha-nodes (for [{:keys [id type alpha-fn children env] :as alpha-map} alpha-fns
                           :let [beta-children (map id-to-node children)]]
-                      [type (with-meta
-                              (eng/->AlphaNode env beta-children alpha-fn)
-                              {:alpha-expr (:alpha-expr (meta alpha-map))})])
+                      [type (eng/->AlphaNode id env beta-children alpha-fn)])
```

Comment:
I was torn here, i was temped to add the AlphaNodes to the `id-to-node` since alpha nodes have ids now.

Reply:
Maybe. I'm not sure what utility there would be in it. It may be useful at some point.

Reply:
Seems like it could be useful for debugging and REPL exploration.

Reply:
Since there seems to be value, i will add them